version/1/4/it/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+# Patto di Condotta dei Collaboratori
+
+## Il Nostro Impegno
+
+Nell'interesse di incoraggiare un ambiente sociale aperto e accogliente, noi in quanto collaboratori e responsabili ci impegniamo a rendere la partecipazione al nostro progetto e alla nostra comunità un'esperienza priva di molestie per tutti, indipendentemente da età, dimensioni del corpo, disabilità, etnia, identità ed espressione di genere, livello di esperienza, nazionalità, aspetto fisico, razza, religione, o identità e orientamento sessuale.
+
+## I Nostri Standard
+
+Esempi di comportamento che contribuiscono a creare un ambiente sociale positivo includono:
+
+* Usare un linguaggio accogliente e inclusivo
+* Essere rispettosi di punti di vista ed esperienze differenti
+* Accettare in modo composto critiche costruttive
+* Concentrarsi su cosa è meglio per la comunità
+* Mostrare empatia verso gli altri membri della comunità
+
+Esempi di comportamento inaccettabile da parte dei partecipanti includono:
+
+* L'uso di linguaggio o immagini sessuali e attenzioni o advances sgraditi
```

Comment:
Actually, in Italian the French plural version is favoured: "attenzioni o avances sgradite" :)